_posts/2017-10-21-roadmap.md

```diff
@@ -4,7 +4,7 @@ title: In Progress
 
 **High-level features we are working on and features we have committed to implementing**
 
-> See our [roadmap on Github](https://github.com/orgs/SoftwareEngineeringDaily/projects/1). If you have an idea for a feature, [email us](mailto:ideas@softwareengineeringdaily.com) or [join Slack](http://softwaredaily.herokuapp.com/)!
```

Comment:
This one is actually correct as is--the brainstorming project board is another representation of the roadmap on Github.

Reply:
Ah no worries then